src/main/java/Ixo.java

```diff
@@ -0,0 +1,151 @@
+                    if (taskToMark >= storeIndex || taskToMark < 0) {
+                        System.out.println("that task does not exist");
+                        System.out.println(SEPARATOR);
+                    }
+                    boolean isMark = cmd.equals("mark");
+                    if (isMark) {
+                        System.out.println("Well done! task " + taskStore[taskToMark] + " has been marked.");
+                    } else {
+                        System.out.println("The task " + taskStore[taskToMark] + " has been unmarked.");
+                    }
+                    taskStore[taskToMark].isDone = isMark;
+                } catch (NumberFormatException e) { //for non-number input
+                    System.out.println("You did not enter a number");
+                } catch (ArrayIndexOutOfBoundsException e) { //for out of bounds index entered
+                    System.out.println("You did not enter a valid corresponding number");
+                }
+                break;
+
+            case "todo":
+                try {
+                    taskStore[storeIndex] = new Todo(inputLine.split("todo ")[1]);
+                    storeIndex++;
+                    System.out.println("Todo added, you have " + storeIndex + " task(s) on the list");
+                } catch (ArrayIndexOutOfBoundsException e) {
+                    System.out.println("You can't add an empty Todo");
+                }
+
+
+                break;
+
+            case "deadline":
+                try {
+                    content = inputLine.split("deadline ")[1].split("/by");
+                    taskStore[storeIndex] = new Deadline(content);
+                    storeIndex++;
+                    System.out.println("Deadline added, you have " + storeIndex + " task(s) on the list");
+                } catch (IndexOutOfBoundsException e) { //for out of bounds index entered
+                    System.out.println("You can't add an empty Deadline");
+                }
+                break;
+
+            case "event":
+                try {
+                    content = inputLine.split("event ")[1].split("/from");
+                    String description = content[0];
+                    String duration = content[1];
+                    taskStore[storeIndex] = new Event(description, duration);
+                    storeIndex++;
+                    System.out.println("Event added, you have " + storeIndex + " task(s) on the list");
+                } catch (IndexOutOfBoundsException e) { //for out of bounds index entered
+                    System.out.println("You can't add an empty Event");
+                }
+                break;
+
+            default:
+                lineScan.nextLine();
+                System.out.println("Unknown command. Type 'bye' to exit");
+                break;
+            }
+        }
+    }
+
+
+    public static void command() {
```

Comment:
Again, the name of this function is not apt. It needs to be a verb. 
```suggestion
    public static void parseCommand() {
```